app/contribute/page.tsx

```diff
@@ -49,17 +49,67 @@ export default function About() {
                 not &quot;anti-AI&quot; &mdash; not by a long shot.
               </li>
               <li>
                 <strong>Reliability</strong> &mdash; Our project relies on
                 well-researched, verifiable instances of harm. We do not accept
                 anecdotal evidence. It is crucial that our catalog of harms is
                 as accurate as possible. More information about our sourcing and
-                verification process is below.
+                verification process is in the next section.
               </li>
             </ol>
+            <h2>Sourcing New Incidents</h2>
+            <p>When finding new incidents to add to Badness.ai, please ensure they meet
+              the following standards designed to uphold the quality and veracity of the
+              catalog.</p>
+            <ul>
+              <li>
+                Badness.ai is a catalog of real-world harms &mdash; demonstrative or not.
+                As a result, &ldquo;jailbreaks&rdquo; and other examples of
+                system vulnerabilities are generally not in scope for the
+                catalog, unless they caused harm or indicate novel
+                potential for harm. For example, the many examples of what people
+                were able to get ChatGPT to say via the
+                infamous <a href="https://www.cnbc.com/2023/02/06/chatgpt-jailbreak-forces-it-to-break-its-own-rules.html">DAN jailbreak</a> are
+                not in scope; these examples would not realistically occur
+                in the wild since they require the user to intentionally jailbreak the
+                underlying model. However, an example where a
+                user <i>unexpectedly</i> jailbreaks the underlying model and
+                then encounters harmful behavior would be in scope for the catalog.
+              </li>
+              <li>
+                When adding new entries to the catalog, please make a best effort to
+                ensure their content is authentic and verifiable. If a source is
+                considered reliable as per <a href="https://en.wikipedia.org/wiki/Wikipedia:Reliable_sources/Perennial_sources">Wikipedia's
+                  crowdsourced standard</a>, then it is generally fine to cite for a
+                catalog entry. For other sources, exercise your best judgement.
+                If someone posts about a harm on Twitter and includes screenshots, it
+                should be fine to include for a catalog entry (assuming it passes your
+                personal sanity check and seems reasonable).
+              </li>
+              <li>
+                Badness.ai is focused on documenting real <i>harms</i> caused by
+                generative AI. The different types of harm the catalog currently
+                covers are available <a href="https://badness.ai/categories">on the categories page</a>.
```

Comment:
Should be a relative link (so that, e.g., it works on localhost)

```suggestion
                covers are available <a href="/categories">on the categories page</a>.
```